TestSwitchApi/TestSwitchApi/Models/DataModels/CandidateTestModel.cs

```diff
@@ -13,7 +13,8 @@ public class CandidateTestModel
         public string TestResult { get; set; }
         public string TestAnswer { get; set; }
 
-        public DateTime StartTime { get; set; }
-        public DateTime EndTime { get; set; }
+        public DateTime? StartTime { get; set; }
+        public DateTime? EndTime { get; set; }
+        public string TestStatus { get; set; }
```

Comment:
I _think_ you'll need to delete this property now :)

Reply:
Property removed